crds/operators.coreos.com_olmconfigs.yaml

```diff
@@ -43,6 +43,9 @@ spec:
                     disableCopiedCSVs:
                       description: DisableCopiedCSVs is used to disable OLM's "Copied CSV" feature for operators installed at the cluster scope, where a cluster scoped operator is one that has been installed in an OperatorGroup that targets all namespaces. When reenabled, OLM will recreate the "Copied CSVs" for each cluster scoped operator.
                       type: boolean
+                    packageServerWakeupInterval:
```

Comment:
Would `packageServerPeriodicSyncInterval` be more appropriate?

Reply:
It's called wakeupInterval in the code... but I really don't care.

Reply:
"Periodic" and "Interval" are redundant. SO, perhaps just "packageServerSyncInterval"?